pb/messages.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package pb;
+
+import "google/protobuf/timestamp.proto";
+
+// Status messages send generic meta information about the state of the inbox.
+message Status {}
+
+// Message objects represent a single, encrypted message sent from one inbox to another.
+message Message {
+    string from = 1;                           // the inbox id of the sender
+    string to = 2;                             // the inbox id of the recipient
+    google.protobuf.Timestamp timestamp = 3;   // the time the message was sent
+    bytes body = 4;                            // the encrypted message body
+}
```

Comment:
Also mentioned in standup - having this much information viewable by the system is a minor security risk - it will allow us to identify who is communicating and how much. We may have to consider the implications of this in the future.